package.json

```diff
@@ -21,11 +21,11 @@
   },
   "homepage": "https://github.com/itsmeow/curseforge-upload#readme",
   "dependencies": {
-    "@actions/core": "^1.10.0",
+    "@actions/core": "^1.10.1",
     "json-schema": "^0.4.0",
     "request": "^2.88.2"
   },
   "devDependencies": {
-    "@zeit/ncc": "^0.22.3"
+    "@vercel/ncc": "^0.38.1"
```

Comment:
zeit is deprecated in favor of vercel, more info can be found on the package page on npm